cmds/graph.c

```diff
@@ -67,7 +67,7 @@ static void print_addr(struct field_data *fd)
 	/* uftrace records (truncated) 48-bit addresses */
 	int width = sizeof(long) == 4 ? 8 : 12;
 
-	pr_out("%*lx", width, node->addr);
+	pr_out("%*llx", width, effective_addr(node->addr));
```

Comment:
I know it's annoying but you should use `PRIx64` for `uint64_t` type.

Reply:
Thanks for pointing this out. I just updated it.